<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriendDialog</class>
 <widget class="QDialog" name="AddFriendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>add friend</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lab_id">
     <property name="text">
      <string>chat id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_id"/>
   </item>
   <item>
    <widget class="QLabel" name="lab_message">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="edit_message"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_send">
     <property name="text">
      <string>send friend request</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
